dependencies.gradle

```diff
@@ -87,7 +80,9 @@ def VERSIONS = [
 def PLATFORM_BOMS = [
         libs.reactorBom,
         libs.nettyBom,
-        libs.junitBom
+        libs.junitBom,
+        libs.prometheusMetricsBom,
+        libs.prometheusSimpleClientBom
```

Comment:
This will end up applying the Prometheus BOMs to all modules, but the Prometheus BOMs should only be needed in the Prometheus modules. Historically we did this kind of global management of dependencies, but I think for new changes, it will be better to keep it more focused where it makes sense and ultimately use this global `dependencies.gradle` less and less. So, I would remove the Prometheus BOMs from here and apply the right one in the corresponding Prometheus module instead.

Reply:
@shakuzen Thanks for the feedback! I updated in 3bea2a1 as suggested.